cognite/client/data_classes/capabilities.py

```diff
@@ -0,0 +1,748 @@
+
+
+@dataclass
+class ExtractionPipelinesAcl(Capability):
+    _capability_name = "extractionPipelinesAcl"
+
+    class Action(enum.Enum):
+        Read = "READ"
+        Write = "WRITE"
+
+    class Scope:
+        All = AllScope
+        ID = IDScope
+        DataSet = DataSetScope
+
+    actions: Sequence[Action]
+    scope: AllScope | IDScope | DataSetScope
+
+
+@dataclass
+class ExtractionsRunAcl(Capability):
+    _capability_name = "extractionRunsAcl"
+
+    class Action(enum.Enum):
+        Read = "READ"
+        Write = "WRITE"
+
+    class Scope:
+        All = AllScope
+        DataSet = DataSetScope
+        ExtractionPipeline = ExtractionPipelineScope
+
+    actions: Sequence[Action]
+    scope: AllScope | DataSetScope | ExtractionPipelineScope
+
+
+@dataclass
+class FilesAcl(Capability):
+    _capability_name = "filesAcl"
+
+    class Action(enum.Enum):
+        Read = "READ"
+        Write = "WRITE"
+
+    class Scope:
+        All = AllScope
+        DataSet = DataSetScope
+
+    actions: Sequence[Action]
+    scope: AllScope | DataSetScope
+
+
+@dataclass
+class FunctionsAcl(Capability):
+    _capability_name = "functionsAcl"
+
+    class Action(enum.Enum):
+        Read = "READ"
+        Write = "WRITE"
+
+    class Scope:
+        All = AllScope
+
+    actions: Sequence[Action]
+    scope: AllScope = field(default_factory=AllScope)
+
+
+@dataclass
+class GeospatialAcl(Capability):
+    _capability_name = "geospatialAcl"
+
+    class Action(enum.Enum):
+        Read = "READ"
+        Write = "WRITE"
+
+    class Scope:
+        All = AllScope
+
+    actions: Sequence[Action]
+    scope: AllScope = field(default_factory=AllScope)
+
+
+@dataclass
+class GeospatialCrsAcl(Capability):
+    _capability_name = "geospatialCrsAcl"
+
+    class Action(enum.Enum):
+        Read = "READ"
+        Write = "WRITE"
+
+    class Scope:
+        All = AllScope
+
+    actions: Sequence[Action]
+    scope: AllScope = field(default_factory=AllScope)
+
+
+@dataclass
+class GroupsAcl(Capability):
+    _capability_name = "groupsAcl"
+
+    class Action(enum.Enum):
+        Create = "CREATE"
+        Delete = "DELETE"
+        Read = "READ"
+        List = "LIST"
+        Update = "UPDATE"
+
+    class Scope:
+        All = AllScope
+        CurrentUser = CurrentUserScope
+
+    actions: Sequence[Action]
+    scope: AllScope | CurrentUserScope
+
+
+@dataclass
+class LabelsAcl(Capability):
+    _capability_name = "labelsAcl"
+
+    class Action(enum.Enum):
+        Read = "READ"
+        Write = "WRITE"
+
+    class Scope:
+        All = AllScope
+        DataSet = DataSetScope
+
+    actions: Sequence[Action]
+    scope: AllScope | DataSetScope
+
+
+@dataclass
+class ModelHostingAcl(Capability):
+    _capability_name = "modelHostingAcl"
+
+    class Action(enum.Enum):
+        Read = "READ"
+        Write = "WRITE"
+
+    class Scope:
+        All = AllScope
+
+    actions: Sequence[Action]
+    scope: AllScope = field(default_factory=AllScope)
```

Comment:
This is not needed. This service was removed a long time ago. I'll update the docs to reflect it.